### ISSUE TITLE
A11y: Non-Descriptive Link Text for Collections, Best Bets, Routed Queries, RSS, and Jobs

### ISSUE THREAD

**Issue by Keisha19881** [GSA/search-gov-website#313]
https://search.gov/get-started/site-launch-guide.html

**Defect:** The link text for the following section......Collections, Best Bets, Routed Queries, RSS, and Jobs is non-descriptive and lacks sufficient context for screen reader users. These links appear to lead to different functionalities within the site, but the text alone does not convey their purpose clearly. 

Suggested fix: "Each of the links should be rewritten to clearly explain its function or destination. For example, replace generic link texts like ""Collections"", ""Best Bets"", etc., with more descriptive phrases.

Example: 
<a href=""collections.html"">Configure alternative search scopes for your site</a>

<a href=""bestbets.html"">Pin search results to the top with Best Bets</a>

<a href=""routedqueries.html"">Bypass search results and route queries directly to a page</a>

<a href=""rss.html"">Integrate RSS feeds into your search results</a>

<a href=""jobs.html"">Show your agency’s job postings in search results</a>

"

![Image](https://github.com/user-attachments/assets/af994414-9071-471b-a20b-b1fdf22ea1a3)
